testing/sdk_cmd.py

```diff
@@ -336,7 +336,7 @@ def _task_exec(task_id_prefix: str, cmd: str) -> tuple:
     return run_raw_cli("task exec {} {}".format(task_id_prefix, cmd))
 
 
-def resolve_hosts(marathon_task_name: str, hosts: list, bootstrap_cmd: str='./bootstrap') -> bool:
+def resolve_hosts(marathon_task_name: str, hosts: list, bootstrap_cmd: str = './bootstrap') -> bool:
```

Comment:
I always thought that spaces around `kwarg` defaults were not recommended?

Reply:
I was getting `flake8` errors on this: `missing whitespace around parameter equals [E252]`